tests/python/unittest/test_sparse_operator.py

```diff
@@ -1447,7 +1458,9 @@ def test_sparse_square_sum():
                     check_numeric_gradient(test, [rsp], grad_stype_dict={'data': 'row_sparse'},
                                            atol=1e-2, rtol=0.1)
 
-                    
+
+# This test takes several minutes to run, marking as nightly.
+@attr('nightly')
```

Comment:
We can't move the sparse tests to nightly. Try reduce the array dimensions.

Reply:
Will do...